target/scripts/check-for-changes.sh

```diff
@@ -72,8 +70,11 @@ function _handle_changes() {
   fi
 
   _ssl_changes
-  _postfix_dovecot_changes
-  _rspamd_changes
+  # For account provisioner LDAP only SSL changes are relevant.
+  if [[ ${ACCOUNT_PROVISIONER} == 'FILE' ]]; then
+    _postfix_dovecot_changes
+    _rspamd_changes
+  fi
```

Comment:
Looks good, just a minor adjustment:
- I'm pretty sure the related comment that was removed at the top of this file was added before Rspamd landed, I think LDAP users can benefit from that too. It's mostly the account file config with Postfix/Dovecot that shouldn't be processed for LDAP.
- I know that LDAP users have configured [relay host support before via the DMS integration](https://docker-mailserver.github.io/docker-mailserver/v15.1/config/advanced/mail-forwarding/relay-hosts/#technical-details), so change detection support there may also be relevant at some point? I've added a TODO comment as I presently lack the time to assess change detection support properly.

```suggestion
  # TODO: Consider support relay host config change support for other provisioners
  if [[ ${ACCOUNT_PROVISIONER} == 'FILE' ]]; then
    _postfix_dovecot_changes
  fi

  _rspamd_changes
```